index.md

```diff
@@ -5,19 +5,32 @@
 layout: home
 ---
 
-The PELUX Base Platform is a Linux platform that is used to kick-start a project for automotive software
-development. The PELUX Base Platform software architecture and design are optimized to secure the shortest path
-from silicon to pixel and thereby provides optimal patterns to deliver a stunning in-vehicle infotainment solution.
+# PELUX, an Automotive Reference Integration Platform
 
-Though PELUX is not limited to infotainment systems. It aims to target other converged automotive systems,
-like the advanced driver-assistance systems (ADAS) and telematics units.
+PELUX is an agnostic reference integration platform that enables you to build
+and develop automotive Linux-based systems without vendor lock-in. As an open
+source solution accelerator, PELUX helps you prototype and create
+production-ready, converged automotive systems, including advanced
+driver-assistance systems (ADAS), telematics units and in-vehicle infotainment
+systems.
 
-PELUX leverages well known open source projects such as Linux, Yocto/OpenEmbedded and GENIVI in order to provide
-a base development platform. The base development platform is complemented by project blueprints and documentation.
+The platform, based on Yocto Linux, is complemented with the Luxoft Software
```

Comment:
```suggestion
The Linux platform, built with Yocto, is complemented with the Luxoft Software
```

Reply:
Even though the factsheet is as this, I will use your suggestion.